bin/p2-shutdown/main.go

```diff
@@ -59,12 +56,12 @@ func main() {
 	var haltWG sync.WaitGroup
 	for _, realityEntry := range reality {
 		pod := podFactory.NewLegacyPod(realityEntry.Manifest.ID())
-		if *dryRun {
-			log.Printf("dry run, skipping this pod: %s", pod.Id)
+		if !shutdownPod(pod.Id, podsToShutdown, *excludePods) {
```

Comment:
nit: `shouldShutdownPod()` would better indicate that this function doesn't perform the down shutting